### PR TITLE
minor fixing, added authorship

### PR DESCRIPTION
**Pull request by Yzkkk**
.

### CODE REVIEW

File: src/main/java/seedu/duke/commands/DeleteCommand.java [hunk 1]
```diff
@@ -47,7 +47,7 @@ public DeleteCommand(int taskIndex, String taskModule) {
 
     @Override
     public CommandResult execute(ModuleList moduleList, Configuration configuration) throws ModHappyException {
-        if (taskIndex < 0) { //If invalid task number or no task number was provided
+        if (!Objects.isNull(moduleCode)) {
             deleteModule(moduleList);
```
Comment (Ch40gRv1-Mu) [inline src/main/java/seedu/duke/commands/DeleteCommand.java:59]: I suggest keep the comments.

File: src/main/java/seedu/duke/commands/EditCommand.java [hunk 2]
```diff
@@ -18,7 +18,7 @@
     private static final String EDIT_TASK_SUCCESS = StringConstants.EDIT_TASK_SUCCESS;
     private static final String EDIT_TASK_WITH_MODULE_SUCCESS = StringConstants.EDIT_TASK_WITH_MODULE_SUCCESS;
     private static final String TASK_DESCRIPTION = StringConstants.TASK_DESCRIPTION_STR;
-    private static final String ESTIMATED_WORKING_TIME = StringConstants.TASK_ESTIMATED_WORKING_TIME_STR;
+    private static final String TASK_ESTIMATED_WORKING_TIME = StringConstants.TASK_ESTIMATED_WORKING_TIME_STR;
```
Comment (Ch40gRv1-Mu) [inline src/main/java/seedu/duke/commands/EditCommand.java:23]: If both are private static final,  should we still declare it repeatedly? How about just use the constant directly.

File: src/main/java/seedu/duke/commands/EditCommand.java [hunk 3]
```diff
@@ -84,7 +85,7 @@ private Module getTargetModule(ModuleList moduleList) throws NoSuchModuleExcepti
 
     @Override
     public CommandResult execute(ModuleList moduleList, Configuration configuration) throws ModHappyException {
-        if (taskIndex < 0) {
+        if (!Objects.isNull(moduleCode)) {
             editModuleDescription(moduleList);
```
Comment (Ch40gRv1-Mu) [inline src/main/java/seedu/duke/commands/EditCommand.java:100]: Same as before, suggests leaves comments explaining the magic logic.

File: src/main/java/seedu/duke/parsers/AddTaskParser.java [hunk 4]
```diff
@@ -21,7 +21,7 @@
     private static final String TASK_ESTIMATED_WORKING_TIME_STR = StringConstants.TASK_ESTIMATED_WORKING_TIME_STR;
     private static final String TASK_NAME = StringConstants.TASK_NAME;
     private static final String TASK_DESCRIPTION = StringConstants.TASK_DESCRIPTION;
-    private static final String TASK_WORKING_TIME = StringConstants.TASK_ESTIMATED_WORKING_TIME;
+    private static final String TASK_ESTIMATED_WORKING_TIME = StringConstants.TASK_ESTIMATED_WORKING_TIME;
```
Comment (Ch40gRv1-Mu) [inline src/main/java/seedu/duke/parsers/AddTaskParser.java:25]: I suggest just use the constants from StringConstants directly without repeated declaration.

**Review by Ch40gRv1-Mu — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/duke/commands/DeleteCommand.java (hunk 1 above)
Comment (Yzkkk) [reply src/main/java/seedu/duke/commands/DeleteCommand.java:59]: The logic is different now. Last time it was "when invalid task number or no task number was provided by the user, we assume that it is not `'del task'` but rather it's a `'del mod [MODULE_CODE]'` input and thus execute the `deleteModule(moduleList)`. Now, it is "execute deleteModule when moduleCode is detected", which I think makes more sense. Also, now the invalid task number/no task number errors have been captured as exceptions when parsing, so it shouldnt appear in DeleteCommand.

File: src/main/java/seedu/duke/parsers/AddTaskParser.java (hunk 4 above)
Comment (Yzkkk) [reply src/main/java/seedu/duke/parsers/AddTaskParser.java:25]: This is to be consistent. We cant have a `StringConstants.TASK_ESTIMATED_WORKING_TIME` for eg within our methods, right?

**Pull request merged**